public/docs/ts/latest/quickstart.jade

```diff
@@ -154,89 +154,90 @@ block install-packages
 
 :marked
   You're now ready to write some code!
-+ifDocsFor('ts')
-  .l-main-section
-  h1#ngmodule Step 2: Create your application
-  :marked
-    You compose Angular applications into closely related blocks of functionality with
-    [NgModules](guide/ngmodule.html). Angular itself is split into separate Angular Modules. This
-    makes it possible for you to keep payload size small by only importing the parts of Angular
-    that your application needs.
-
-    Every Angular application has at least one module: the _root module_, named `AppModule` here.
-
-    **Create #{_an} #{_appDir} subfolder** off the project root directory:
-
-  code-example.code-shell.
-    mkdir #{_appDir}
-
-  :marked
-    Create the file `app/app.module.ts` with the following content:
-
-  +makeExample('app/app.module.1.ts')(format='.')
-
-  :marked
-    This is the entry point to your application.
-
-    Since the QuickStart application is a web application that runs in a browser, your root module
-    needs to import the
-    [`BrowserModule`](../latest/api/platform-browser/index/BrowserModule-class.html)
-    from `@angular/platform-browser` to the `imports` array.
-
-    This is the smallest amount of Angular that is needed for a minimal application to run in the
-    browser.
-
-    The QuickStart application doesn't do anything else, so you don't need any other modules. In a real
-    application, you'd likely import [`FormsModule`](../latest/api/forms/index/FormsModule-class.html) 
-    as well as [`RouterModule`](../latest/api/router/index/RouterModule-class.html) and
-    [`HttpModule`](../latest/api/http/index/HttpModule-class.html). These are introduced in the
-    [Tour of Heroes Tutorial](./tutorial/).
 
 .l-main-section
 h1#root-component Step !{step++}: Create a component and add it to your application
 :marked
   Every Angular application has at least one component: the _root component_, named `AppComponent`
-  here.
+  by convention.
 
```

Comment:
>What a developer calls the root component is their call.
The docs, the styleguide and the cli all create and refer to `AppModule` and `AppComponent`. These are not _my_ conventions. They are _our_ conventions. 

As with _any_ conventions you are free to create your own, but you would be wise to do so only with a good reason.